web_footer_gdpr_report/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Web Footer GDPR Report",
+    "summary": """
+        Adds footer GDPR report in layout web
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Sale","web"
+    "website": "https://github.com/solvosci/slv-report",
+    "depends": [
+        "sale","web"
+    ],
+    "data": [
+        "views/res_company_views.xml",
```

Comment:
Since the actual model used in views has changed from initial versions, file should be renamed:
```suggestion
        "views/res_config_settings_views.xml",
```